app/containers/scenes/MainMapScene.js

```diff
@@ -200,8 +200,12 @@ export default class MainMapScene extends Component {
 			},
 			(error) => {
 				this.setState({
+					//hard coded value - to avoid error - change this!
```

Comment:
Not a big deal, bit I would have put `TODO` in that comment so that it's easy to search for in the future.